src/org/loklak/harvester/Post.java

```diff
@@ -0,0 +1,47 @@
+package org.loklak.harvester;
+
+import org.json.JSONObject;
+import org.loklak.data.DAO;
+import java.net.URL;
+import java.util.Date;
+import java.util.Map;
+
+/**
+ * @author vibhcool (Vibhor Verma)
+ * @version 0.1
+ * @since 07.06.2017
+ *
+ * Post abstract class for data objects.
+ */
+public abstract class Post extends JSONObject {
```

Comment:
@vibhcool: I wanted to know about some examples of objects which can be `Post`s.

For example, profile of a user can be a `Post`, similarly what else?